doc/Roadmap.rst

```diff
@@ -1,158 +1,156 @@
+       <https://en.wikipedia.org/wiki/Cladogram>`__
+    -  `radial trees <https://en.wikipedia.org/wiki/Radial_tree>`__
+    -  `packed bubbles
+       <https://stackoverflow.com/questions/46131572/making-a-non-overlapping-bubble-chart-in-matplotlib-circle-packing>`__
+    -  `funnel charts <https://en.wikipedia.org/wiki/Funnel_chart>`__
+    -  `donut charts <https://datavizcatalogue.com/methods/donut_chart.html>`__
+       (which can be abused to make a pie chart if you really want)
+
+7.  **Interactive Matplotlib plotting**: Right now, HoloViews supports Matplotlib
+    primarily as static PNG or SVG output. Matplotlib also supports interactive
+    web-based plotting via ipympl, and supporting such interactivity could help
+    make Matplotlib be a viable alternative for more use cases. 
+
+8.  **Better native GUI support**: Right now, HoloViz focuses exclusively on
+    tools that work well in web browsers, because it aims to support the entire
+    workflow from initial exploration to delivery of fully functional interactive
+    applications to other stakeholders. One consequence is that it doesn't
+    currently support the types of interactivity provided by Matplotlib when used
+    with its native GUI backends like Qt. It should be feasible to make the
+    interactive functionality in HoloViews (e.g. streams and DynamicMaps) support
+    native GUI apps as well, allowing the same user code to be applied to a wider
+    range of use cases.
+
+9.  **Altair/Vega/Vega-lite integration**: HoloViews and Bokeh provide
+    declarative syntaxes that can be expressed in purely static form, and it
+    should be feasible to write a translator between them and other declarative
+    libraries like Altair, Vega, and Vega-Lite. Those libraries are currently
+    focused primarily on statistical visualizations, not covering areas like
+    multidimensional (image-like) plotting where HoloViews and Bokeh are also
+    used, but they offer a common interchange format that could be helpful for
+    interoperating with other tools. Writing an import and export facility that
+    covers the bulk of the shared functionality should not be a major undertaking
+    and could open up interesting new applications. For now, Altair and Vega-lite
+    visualizations can be specified and then used directly with Panel, but not
+    with HoloViews.
+
+10. **hvPlot/HoloViews serialization**: HoloViews uses a declarative design
+    that can be represented in a purely textual form, without any Python code. An
+    initial implementation allows any Param-based objects (including HoloViews
+    objects) to be represented in JSON or YAML, but it needs some polishing
+    before it can be put into wide use for saving and restoring configurations
+    and layouts.
+
+11. **Multiple selection groups**: HoloViews currently supports a single
+    selection group. Multiple selection groups would allow for more flexible use
+    of, and comparison between, selections.
```

Comment:
I think this has to be explained better, as I don't think "selection group" is a common term in our docs.

Reply:
text revised